src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Extensions/ProjectExtensions.cs

```diff
@@ -16,10 +16,18 @@ namespace Microsoft.CodeAnalysis.Shared.Extensions
     internal static partial class ProjectExtensions
     {
         public static TLanguageService? GetLanguageService<TLanguageService>(this Project? project) where TLanguageService : class, ILanguageService
+#if CODE_STYLE
             => project?.GetExtendedLanguageServices().GetService<TLanguageService>();
```

Comment:
code style does some funky stuff that we don't need in OOP.  temporarily splitting this behavior so we don't go throught eh workspace here.